pytorch_blade/benchmark/TorchBench/test_torch_bench.sh

```diff
@@ -17,19 +17,20 @@ python3 -m pip install -q -r $script_dir/requirements.txt
 
 # setup for torchbenchmark
 benchmark_repo_dir=$HOME/.cache/torchbenchmark
-if [ -d $benchmark_repo_dir ]; then
-    rm -rf $benchmark_repo_dir
+if [ ! -d $benchmark_repo_dir ]; then
+    git clone -q https://github.com/pai-disc/torchbenchmark.git --recursive $benchmark_repo_dir
```

Comment:
Shall we update benchmark_repo if it's already exsited?

Reply:
Yes, we update after cd into the bench repo.